tests/CMakeLists.txt

```diff
@@ -22,6 +22,9 @@ function(CREATE_PODIO_TEST sourcefile additional_libs)
   target_link_libraries(${name} TestDataModel ${additional_libs})
   set_property(TEST ${name} PROPERTY ENVIRONMENT
       LD_LIBRARY_PATH=${CMAKE_CURRENT_BINARY_DIR}:${CMAKE_BINARY_DIR}/src:$ENV{LD_LIBRARY_PATH}
+      # Clear the ROOT_INCLUDE_PATH for the tests, to avoid potential conflicts
+      # with existing headers from other installations
+      ROOT_INCLUDE_PATH=
```

Comment:
Do you actually have to clear it or will it be enough to prefix it with the correct path?

Reply:
It would in principle work with prefixing `${CMAKE_SOURCE_DIR}/tests/datamodel`. But my thought was that clearing it gives a simpler test environment in the end, and this would be one less thing to worry about.